spec/support/bulk_upload/sales_log_to_csv.rb

```diff
@@ -0,0 +1,187 @@
+class BulkUpload::SalesLogToCsv
+  attr_reader :log, :line_ending, :col_offset, :overrides
+
+  def initialize(log:, line_ending: "\n", col_offset: 1, overrides: {})
+    @log = log
+    @line_ending = line_ending
+    @col_offset = col_offset
+    @overrides = overrides
+  end
+
+  def row_prefix
+    [nil] * col_offset
+  end
+
+  def to_2022_csv_row
+    (row_prefix + to_2022_row).flatten.join(",") + line_ending
+  end
+
+  def default_2022_field_numbers
+    (1..134).to_a
```

Comment:
Not relevant to this ticket but I think the number of fields for 2223 sales is either 125 or 128?

Reply:
ah good spot, you are correct it looks to be `125`, i will get that changed

Reply:
that's done now